src/packet/public_key_encrypted_session_key.js

```diff
@@ -105,8 +105,9 @@ class PublicKeyEncryptedSessionKeyPacket {
       util.writeChecksum(this.sessionKey)
     ]);
     const algo = enums.write(enums.publicKey, this.publicKeyAlgorithm);
+    const privateParams = algo === enums.publicKey.aead ? key.privateParams : {};
```

Comment:
Rather than passing an empty object, I think it might be good to pass `null` if possible, so that any attempt to access a private parameter in one of the other algorithms immediately fails. In addition, it should be slightly more efficient, since this object then doesn't have to be constructed. (Similarly everywhere else.)